include/pq_table.h

```diff
@@ -5,6 +5,8 @@
 
 #include "utils.h"
 
+#define NUM_PQ_CENTROIDS 256
+
```

Comment:
Is this an upper bound on the number of PQ centroids? Then should we name it MAX_PQ_CENTROIDS?